pkg/utils/ethutils.go

```diff
@@ -0,0 +1,66 @@
+package utils
```

Comment:
Is this code from somewhere else? please add a reference if yes.

Reply:
Done